docs/source/index.rst

```diff
@@ -0,0 +1,24 @@
+.. Indy Plenum documentation master file, created by
+   sphinx-quickstart on Mon Nov 26 15:42:54 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Indy Plenum's documentation!
+=======================================
+
+.. toctree::
+   :maxdepth: 2
+
```

Comment:
I think the following order would be more convenient:
    main.md
    glossary.md
    diagrams/index.rst
    request_handling.md
    storage.md
    catchup.md
    message_requests.md
    plugins.md
    start_nodes.md
    recorder.md
    misc/zeromq_features.md